cmd/api/main.go

```diff
@@ -35,17 +37,34 @@ func main() {
 	ctx, logger := belogger.InitLogger(ctx, cfg.Log, "block_explorer_api")
 	logger.Info("Config and logger were initialized")
 
+	router := api.NewRouter()
+	_ = router.Start(ctx)
+	defer func() {
```

Comment:
`Defer` will not be called in the case of `Fatal`

Reply:
oh. I didn't pay attention